vmdb/app/models/miq_report_result.rb

```diff
@@ -95,9 +95,9 @@ def html_rows(options={})
       options[:offset] = ((page - 1) * per_page)
       options[:limit]  = per_page
     end
-    self.update_attribute(:last_accessed_on, Time.now.utc)
-    self.purge_for_user
-    self.html_details.all(options.merge(:order => "id asc")).collect(&:data)
+    update_attribute(:last_accessed_on, Time.now.utc)
```

Comment:
I didn't even read that line.. just dropped the `self.` to avoid angering rubocop. :smile:

But yeah, I think you're thinking of `update_column`.